docs/content/migrate/nginx-to-traefik.md

```diff
@@ -0,0 +1,615 @@
+---
+title: "Migrate from NGINX Ingress Controller to Traefik"
+description: "Step-by-step guide to migrate from Kubernetes NGINX Ingress Controller to Traefik with zero downtime and annotation compatibility."
+---
+
+# Migrate from NGINX Ingress Controller to Traefik
+
+How to migrate from NGINX Ingress Controller to Traefik with zero downtime.
+{: .subtitle }
+
+---
+
+!!! danger "NGINX Ingress Controller Retirement"
+
+    The Kubernetes NGINX Ingress Controller project has announced its retirement in **March 2026**. After this date:
+
+    - No new releases or updates
+    - No security patches
+    - No bug fixes
+
+    For more information, see the [official Kubernetes blog announcement](https://kubernetes.io/blog/2025/11/11/ingress-nginx-retirement).
+
+## What You Will Achieve
+
+By completing this migration, your existing Ingress resources will work with Traefik without any modifications. The Traefik Kubernetes Ingress NGINX Provider automatically translates NGINX annotations into Traefik configuration:
+
+```yaml tab="Your Existing Ingress (No Changes Needed)"
+
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: myapp
+  annotations:
+    # These NGINX annotations are automatically translated by Traefik
+    nginx.ingress.kubernetes.io/ssl-redirect: "true"
```

Comment:
Without `force-ssl-redirect` the redirection will be sent even if a valid certificate doesn't exist:
```suggestion
    nginx.ingress.kubernetes.io/ssl-redirect: "true"
    nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
```